main.md

```diff
@@ -510,17 +516,28 @@ Verifier Service UUID for SCAN_RESP MUST be `00000002-5026-444A-9E0E-D6F2450F3A7
 
 # Security Considerations {#security}
 
+## Desired Security Properties
+
+This specification aims to provide three security properties:
+
+* Secrecy: Wallets never leak verifiable presentation tokens sent.
+* Authentication: Whenever a verifier accepts a verifiable presentation token, it was sent by an honest wallet and that wallet sent the token to that verifier.
+* Replay protection: Every verifiable presentation token is accepted exactly once.
+
 ## Session Information
 
 Both wallet and the Verifier MUST remove all the information about the session after its termination.
 
 ## Verifier Authentication
 
-How does the wallet authenticate the Verifier? The verifier signs the presentation request. 
+The wallet authenticates the verifier by verifying the signature of the presentation request.
+We assume that the wallet has been configured with an authentic public key of the verifier in advance.
```

Comment:
If the wallet is online it could download and cache the key for future usage. Also, we may have an X509 certificate for a verifier who wants to support extensive offline scenarios. A set of root keys could be trusted and the chain can be validated successfully. This is most often true in case of national id & social benefits delivery in remote areas.

Reply:
Went for something in between! I agree it makes sense to explicate possibilities on how to achieve this, but I think it is better to separate assumptions from how to achieve them :)